gapii/cc/call_observer.h

```diff
@@ -41,10 +41,8 @@ class SpyBase;
 // CallObserver collects observation data in API function calls. It is supposed
 // to be created at the beginning of each intercepted API function call and
 // deleted at the end.
-class CallObserver {
+class CallObserver : public context_t {
```

Comment:
Because the we require that this has no vtable (otherwise it would break context_t) can we mark this as 
__declspec(novtable) on windows just so that we can get errors if something goes wrong. Unfortunately something equivalent does not exist for gcc.
Not a huge priority, but might help catch future issues.

Reply:
Would like to do this in follow up CLs.

Reply:
Sounds good to me.